app/segments/drill/[id]/submission/input.js

```diff
@@ -580,11 +694,19 @@ export default function Input({ drillInfo, setToggleResult, setOutputData }) {
   };
 
   //Loading until an attempt is generated
-  if (attemptShots.length === 0) {
+  if (attemptShots.length === 0 || leaderboardIsLoading) {
     console.log("Loading");
     return <Loading />;
   }
 
+  if (leaderboardError) {
+    return (
+      <ErrorComponent
+        message={[userError, drillError, attemptError, leaderboardError]}
```

Comment:
userError, drillError, and attemptError don't exist

Reply:
removed those